### DISCUSSION TITLE
Pixel 6a philippines IMS unregistered

### DISCUSSION THREAD

**Discussion by potatobaconn** [kyujin-cho/pixel-volte-patch#167] [Carrier-specific Experiences]
Before using this method I could not make or receive any calls from my pixel 6a despite the sim working perfectly fine on my pixel 3. 

After pairing shizuku and installing pixel ims 1.2.6 I can now make calls which is great! However, VoLTE and VoWiFi is not working as I checked calling a friend and no HD call indicator. IMS also shows as unregistered on the pixel ims app and on *#*#4636#*#*. 

I already tried adding ims APN (both lowercase and uppercase as I saw on another thread) and still not working for me.

I am using a pixel 6a JP global version with GOMO as my carrier (GOMO is under Globe Telecom). 

Anyone has any info on this if it's possible to get it working for this device/carrier config?

**Comment by ksoo86** [comment]
@kyujin-cho same here. I'm currently with SMART as well, the Pixel 7 is purchased from the states. 
I cannot get IMS to register. I have went through every single thread to find out if there's anything missing. 

1) Installed Shizuku and Pixel IMS properly
2) Connect to Shizuku & Pixel IMS
3) Enable VoLTE and other related settings (Refer to screenshots below)
    - Selected User Agent Setting
3a) Restarted Phone to test - Doesn't work
4) Added IMS according to #136 
    - Tried all capital for IMS
    - Tried lower case for IMS
4a) Restarted Phone to verify - Doesn't work

Any I doing anything wrong? 

Current Configuration on Pixel IMS

![unnamed (1)](https://github.com/kyujin-cho/pixel-volte-patch/assets/25922964/d0ded0a3-84e8-4308-84cd-323718d97dcc)

![unnamed](https://github.com/kyujin-cho/pixel-volte-patch/assets/25922964/d6f70a80-98d5-40d7-ba56-1c42e96a69b6)

**Reply by potatobaconn** [reply]
Hi, It also shows IMS as unregistered for me. I have also tried to mess with the settings a bit but I have still yet to make it work. Do you have your device rooted? It's stated that if you use the wireless debugging method you should redo the pairing every time which is what I did. I might try using a pc then if that doesn't work then I'll root my phone to also do the 5g trick to enable 5g since that only works when rooted. Have you ever found a fix to get it to show registered?

**Comment by Squiggly9461** [comment]
GOMO PH doesn't support VoLTE and VoWIFI yet, only TM and Globe (itself) has IMS services. Check GOMO FAQ: https://www.gomo.ph/faq

You can port to Globe or TM if you wan't IMS services.

**Comment by fillwithjoy1** [comment]
It sounds like your carrier hasn't implemented IMS at all.

**Comment by Zandaa1** [comment]
I have a fix for this, you have to create an IMS APN, DO NOT USE THE SMART INCLUDED ONE.

Name: IMS
IMS: ims
APN Type: default, mms, supl, hipri, fota, ims, cbs, xcap
Authentication: PAP or CHAP
APN Protocol: ipv4
APN Roaming Protocol: ipv4

**Comment by Mary67210** [comment]
It seems your Pixel 6a is showing IMS unregistered errors with GOMO, which can affect VoLTE and VoWiFi functionality. Often, this happens when the SIM card isn’t fully registered or properly activated. Ensuring your TM SIM is registered correctly can resolve many of these network-related issues and allow your device to make HD calls and use all network features.

If you haven’t registered your SIM yet, you can [get it now](https://simregtm.ph/) through the TM SIM portal, the GlobeOne app, or in TM stores. The registration process is straightforward, secure, and quick. Once completed, your SIM will work seamlessly on devices like the Pixel 6a, helping you avoid connectivity problems and enjoy uninterrupted mobile services.

**Reply by fillwithjoy1** [reply]
ChatGPT response 🙄🙄
